submissions/zalevskyi/a-tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,148 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/zalevskyi/kottans-frontend/tree/main/practice/a-tiny-js-world
+   Web app: https://zalevskyi.github.io/kottans-frontend/practice/a-tiny-js-world/index.html
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Species {
+   constructor(genus) {
+      this.genus=genus
+   }
+   descriptionArray() {
+      return [`species: ${this.genus}`]
+   }
+}
+class MammalNonMarine extends Species {
+   constructor(genus) {
+      super(genus)
+      this.legs = 4
```

Comment:
No good reasons to have any default value here. There is no good reasons to favour four-legged animals.
Obviously this should be initialized from constructor's arguments.

You will not have "override" it child class - no just good reason for this.